access-control-app/int-test/cmr/access_control/int_test/acl_crud_test.clj

```diff
@@ -321,8 +321,8 @@
     (are3 [re acl]
           (is (thrown-with-msg? Exception re (ac/create-acl (u/conn-context) acl {:token token})))
 
-          ;; Acceptance criteria: I receive an error if creating an ACL missing required fields.
-          ;; Note: this tests a few fields, and is not exhaustive. The JSON schema handles this check.
+          ; Acceptance criteria: I receive an error if creating an ACL missing required fields.
```

Comment:
Not sure if this change was intentional - I think we're still standardizing on two semicolons, though don't remember the rationale anymore.

Reply:
Should be `;;` -- part of the Clojure style guide (inherited from CL).
* 4 at top of namespace (almost never used in clj due to namespace docstrings)
* 3 for module-level code comments
* 2 for function-level code comments
* 1 for end-of-line comments

Reply:
Note that for uses of Emacs with Lisp plugins, comment lines that start with a single semi-colon can cause issues. I think I remember @rvirding running into this in LFE ...

Reply:
I used atom to comment out a bunch of stuff, then uncommented it.  I think thats why this happened.